src/lib.rs

```diff
@@ -274,6 +274,7 @@ pub fn make_test(config: &Config, testpaths: &TestPaths) -> test::TestDescAndFn
             name: make_test_name(config, testpaths),
             ignore: early_props.ignore,
             should_panic: should_panic,
+            allow_fail: false,
```

Comment:
I tend to agree that fixing this should take priority over implementing new features, because _without_ this fix this crate is not usable right now.

Maybe a separate ticket can be opened to make this configurable?